tools/install.sh

```diff
@@ -342,21 +344,23 @@ setup_zshrc() {
       return
     fi
     
-    # Ask user for confirmation before backing up and overwriting
-    echo "${FMT_YELLOW}Found ${zdot}/.zshrc."
-    echo "The existing .zshrc will be backed up to .zshrc.pre-oh-my-zsh if overwritten."
-    echo "Make sure your .zshrc contains the following minimal configuration if you choose not to overwrite it:${FMT_RESET}"
-    echo "----------------------------------------"
-    cat "$ZSH/templates/minimal.zshrc"
-    echo "----------------------------------------"
-    printf '%sDo you want to overwrite it with the Oh My Zsh template? [Y/n]%s ' \
-      "$FMT_YELLOW" "$FMT_RESET"
-    read -r opt
-    case $opt in
-      [Yy]*|"") ;;
-      [Nn]*) echo "Overwrite skipped. Existing .zshrc will be kept."; return ;;
-      *) echo "Invalid choice. Overwrite skipped. Existing .zshrc will be kept."; return ;;
-    esac
+    if [[ $OVERWRITE_CONFIRMATION != "no" ]]; then
```

Comment:
For the installer we should keep to using only POSIX style conditionals, with single brackets `[`. Some `sh` don't support this syntax.